src/con_duct/__main__.py

```diff
@@ -126,14 +127,29 @@ class ProcessStats:
     rss: int  # Memory Resident Set Size in Bytes
     vsz: int  # Virtual Memory size in Bytes
     timestamp: str
+    etime: str
+    cmd: str
 
     def max(self, other: ProcessStats) -> ProcessStats:
+        cmd = self.cmd
+        if self.cmd != other.cmd:
+            lgr.debug(
+                f"cmd has changed. Previous measurement was {self.cmd}, now {other.cmd}."
+            )
+            # Brackets indicate that the kernel has substituted an abbreviation.
+            surrounded_by_brackets = r"\[.*?\]"
+            if re.search(surrounded_by_brackets, self.cmd):
+                lgr.debug(f"using {other.cmd}.")
+                cmd = other.cmd
```

Comment:
could we just keep that original one in?  might be interesting later for analysis etc.  It would be just something to remember that we need to go by PID (those IIRC shouldn't be readily reused so shouldn't overlap for different commands in consecutive records)

Reply:
We can't keep both with the current data structure-- both measurements are being aggregated into a single sample.

If the cmd has changed for real (with an exec or something?), we can still have different commands with the same pid in subsequent records. We just can't have 2 cmds in the same sample.